Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM golang AS builder
-
-COPY . /go/src/github.com/rclone/rclone/
-WORKDIR /go/src/github.com/rclone/rclone/
-
-RUN \
-  CGO_ENABLED=0 \
-  make
-RUN ./rclone version
-
-# Begin final image
-FROM alpine:latest
-
+ARG ALPINE_VERSION=latest
+ARG GO_VERSION=latest
+ARG XCPUTRANSLATE_VERSION=v0.6.0
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=${BUILDPLATFORM} qmcgaw/xcputranslate:${XCPUTRANSLATE_VERSION} AS xcputranslate
+
+FROM --platform=${BUILDPLATFORM} golang:${GO_VERSION}-alpine${ALPINE_VERSION} AS builder
+COPY --from=xcputranslate /xcputranslate /usr/local/bin/xcputranslate
+RUN apk --update add git make bash
+ENV CGO_ENABLED=0
+WORKDIR /tmp/gobuild
```

Comment:
FYI changed the workdir to `/tmp/gobuild` to highlight it's not required to build from `/go/src/github.com/rclone/rclone/` since we use go modules. I can revert or change to another directory if you prefer.

Reply:
That is fine. Building in `/go/src/github.com/rclone/rclone/` is a leftover from pre-modules days.